pubsubx/kgox/consumer.go

```
@@ -87,9 +87,13 @@ func (c *consumer) bootstrapClient() error {
 	if err != nil {
 		c.l.WithError(err).Warnf("event retry mechanism might not work properly")
```

Personnally, I would return an error here